tests/conftest.py

```diff
@@ -16,3 +17,27 @@ def test_ops(tmpdir, data_dir):
     ops['data_path'] = [data_dir]
     ops['save_path0'] = str(tmpdir)
     return ops
+
+
+@pytest.fixture(scope="module")
```

Comment:
This fixture is very complex for so-specific a problem; it seems to me that it's a sign that the problem it is working around is a bug (or, to put it another way, a missing feature) in the source code.  A simpler approach would be helpful that way.